packages/react-instantsearch-core/src/connectors/connectInfiniteHits.js

```diff
@@ -29,6 +29,27 @@ function getCurrentRefinement(props, searchState, context) {
   return currentRefinement;
 }
 
+function getStateWithoutPage(state) {
+  const { page, ...rest } = state;
+  return rest;
+}
+
+function getInMemoryCache() {
+  let cachedHits = undefined;
+  let cachedState = undefined;
+  return {
+    read: ({ state }) => {
```

Comment:
```suggestion
    read({ state }) {
```

Reply:
Any reason?

Reply:
arrow functions add a layer of .bind in babel in some cases IIRC. At least as a regular function there's less characters

Reply:
https://github.com/algolia/react-instantsearch/pull/2921/commits/0f8bf1ef036d53495b29645439ec2e27b91d9737